.circleci/config.yml

```diff
@@ -117,11 +146,42 @@ jobs:
             - ~/.mix
       - run: mix dialyzer --halt-exit-status
 
+  release:
+    working_directory: ~/ethereum
+    docker:
+      - image: elixir:latest
```

Comment:
We could also, similarly, produce an artifact here?

Reply:
Good point! For now I've added a commit that hardcodes the release version because we haven't had a discussion on how do we plan to generate releases. The release gets uploaded.